packages/application-components/src/components/modal-page/internals/modal-page-top-bar.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Text,
+  CloseIcon,
+  Spacings,
+  FlatButton,
+  AngleLeftIcon,
+  SecondaryIconButton,
+} from '@commercetools-frontend/ui-kit';
+import { getTopBarStyles } from './modal-page.styles';
+
+const ModalPageTopBar = props => (
+  <div css={getTopBarStyles(props)}>
+    <Spacings.Inline>
+      <FlatButton
```

Comment:
from the designs it looks like there can be an array of previous paths right?

Reply:
At first I thought so, like breadcrumbs, but then I didn't find any other than just the previous path and the current one. I'm going to clarify this.

Reply:
Actually, on another look I've found sometimes there are more than just one previous paths, so I'm going to refactor this to use an array for such.

Reply:
After discussing with design, this is only supposed to act like a "Back Button". There are plans to implement a "breadcrumbs" component but not for this phase.